native/fastlane/Fastfile

```diff
@@ -108,30 +108,33 @@ platform :android do
   # apk_path: The path of the apk to upload (relative to home dir)
   desc "Deliver the app to Play Store. Depending on the option `production_delivery` the update is released to the general public."
   lane :playstore_upload do |options|
+    ensure_env_vars(
+        env_vars: ["GOOGLE_SERVICE_ACCOUNT_JSON"]
+    )
+
     version_code = options[:version_code]
     version_name = options[:version_name]
     build_config_name = options[:build_config_name]
     apk_path = options[:apk_path]
 
     build_config = read_build_config(build_config_name)
-
-    ensure_env_vars(
-        env_vars: ["GOOGLE_SERVICE_ACCOUNT_JSON"]
-    )
-
+    skip_metadata = build_config_name == "malte"
```

Comment:
I'd favor to have this check in the circleci files. The fastline files do not need to know about which build configs are available.

Reply:
This is not about which build configs are available but rather for which build configs metadata is existing. Metadata is only handled in fastlane, so I think this is the clearer way. Other opinions?

Reply:
🙋🏼‍♀️ why do we skip the metadata if the build config is malte?

Reply:
If we upload all metadata on every delivery, it is not possible to change metadata in the playstore console or in appstore connect, so we would have to change it in our repo. Since we don't want to do that for the Maltesers and allow them to do it themselves, we don't upload any metadata.

Reply:
Thanks for explaining. I agree the check should remain in the fastline files. It does seem like a realtively elegant solution to me.